packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/TreeActionButton.tsx

```diff
@@ -32,10 +40,19 @@ export type FilterActionProps = {
 } & Partial<Pick<ReturnType<typeof useTree>, "getHierarchyLevelDetails">>;
 
 /** @internal */
-export function TreeActionButton({ show, label, action, icon }: TreeItemAction) {
+export function TreeActionButton({ show, label, action, icon, activeDescription }: TreeItemAction) {
   const { localizedStrings } = useLocalizationContext();
   const localizedLabel: string | undefined = localizedStrings[label as keyof typeof localizedStrings];
-  return <Tree.ItemAction visible={show} onClick={action} label={localizedLabel ?? label} icon={icon ?? placeholderIcon} />;
+  const localizedDescription: string | undefined = localizedStrings[activeDescription as keyof typeof localizedStrings];
```

Comment:
I don't understand this. The docs for `activeDescription` prop say that it's alreadya user-friendly value that we can show, but here it's being used as a key for `localizedStrings`. Can you explain?

Reply:
We provide functions for some buttons (filter, visibility), they need descriptions we can allow passing in descriptions when creating the button or we can provide a localizedString value. Because you would only want to change the description on different language, I believe using localized strings is better.

Reply:
If we are sure consumers are going to use `createUiItemsProviders` or `getWidgets` in a way that allows hooks i'm fine with it.

Reply:
They will, most likely, not going to call them from render functions. However, they can always render their own custom component that can use hooks and then render our trees.

Reply:
Changed

Reply:
Looks good. Please add a changefile mentioning the `createFilterAction` -> `useFilterAction` change.